ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -370,6 +370,9 @@ export default class ConfirmTransactionBase extends Component {
   updateValueToMax() {
     const { maxValue: value, txData, updateTransaction } = this.props;
 
+    // Remove status from txData to avoid overwriting the txData.status
+    delete txData.status;
```

Comment:
My understanding was that `updateTransaction` assigns the entire property in the state array meaning we always have to provide the full transaction data? Would this not result in a missing status?

Would it be safer to instead use `updateEditableParams` so we can update only the `value` and nothing else?

Reply:
I thought we were merging with previous one on update, I was totally wrong, now it raises some question on my head that what if we send partial `TransactionMeta` to `updateTransaction`. We always wanted to make atomic updates with different update methods but this made me feel a bit concerned now. 

I didn't even see an error (losing status) while doing manual tests too which is weird, if I followed properly it's not merged in redux either. 

Anyway, you're right, it would make sense to make it smallest update. I added a method to it's container just to update it's value.